crypto/s390xcpuid.pl

```diff
@@ -129,6 +129,14 @@
 .type	OPENSSL_rdtsc,\@function
 .align	16
 OPENSSL_rdtsc:
+	larl	%r4,OPENSSL_s390xcap_P
+	tm	S390X_STFLE+3(%r4),0x40	# check for store-clock-fast facility
+	jz	.Lstck
+
+	stckf	16($sp)
```

Comment:
Related question here is why are km instructions coded with storage directives such as .long? Because we don't want to make assumption about assembler version. Because older assemblers wouldn't recognize mnemonics and fail. Yes, one can totally argue that it might be time to let it go, but then one ought to look around, formulate new boundary condition (which is new minimum assembler version that we don't let fail), and harmonize all code. And this request is probably not right place to do that. So suggestion here is to
```
.long 0xb27cf010 # stckf 16($sp)
```

Reply:
Just in case, current minimum version is 2.15.something.

Reply:
Your right, changed to hex encoding.